backend/cms/models/page.py

```diff
@@ -86,10 +95,14 @@ class PageTranslation(models.Model):
     text = models.TextField()
     language = models.ForeignKey(Language, on_delete=models.CASCADE)
     version = models.PositiveIntegerField(default=0)
-    public = models.BooleanField(default=False)
+    choices = (
+        (True, 'Öffentlich'),
+        (False, 'Privat')
+    )
```

Comment:
`choices` is not an appropriate name. A boolean field `public` is just fine imo.